src/components/common/Stepper.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <v-container id="step-buttons-container" class="list-item">
+
+    <v-btn id="step-1-btn" large to="/define-company">
+      <span>Define Your<br>Company</span>
+    </v-btn>
+
+    <v-btn id="step-5-btn" large to="/review-confirm">
+      <span>Review<br>and Confirm</span>
+    </v-btn>
+
+  </v-container>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Watch } from 'vue-property-decorator'
+
+@Component
+export default class Stepper extends Vue {
+  @Watch('$route', { immediate: true })
```

Comment:
What does the option `immediate: true` do?

Reply:
You get an initial event (before the value changes).

There's also an option `deep: boolean` that generates an event when an object's properties change.

Reply:
interesting, thanks!